### ISSUE TITLE
Error "Linking a Tuya device with Smart Link"

### ISSUE THREAD

**Issue by diskxo** [codetheweb/tuyapi#425]
![image](https://user-images.githubusercontent.com/64780086/108128552-32de1380-70ad-11eb-8125-aff31473573d.png)
I receive this error when I try to apply the
4th point of "Linking a Tuya device with Smart Link". What I should do?

**Comment by codetheweb** [comment]
What happens if you click the Verify button?

**Comment by diskxo** [comment]
> What happens if you click the Verify button?

it links the account: https://iot.tuya.com/user/account/info

![image](https://user-images.githubusercontent.com/64780086/108402625-131a2d00-721e-11eb-8c6a-30eb1930e802.png)

https://iot.tuya.com/user/account/enterprise

![image](https://user-images.githubusercontent.com/64780086/108402694-23caa300-721e-11eb-8d02-d0f7aee44c6f.png)
https://support.tuya.com/en/help/_detail/K8sdybj71dz4w

![image](https://user-images.githubusercontent.com/64780086/108402726-2e853800-721e-11eb-9386-2d562c71c563.png)

**Comment by diskxo** [comment]
In my account the voice: "Verification Status" is on "Unverified identity", https://user-images.githubusercontent.com/64780086/108402625-131a2d00-721e-11eb-8c6a-30eb1930e802.png , is this permission required to use the API? because without that I can't use anything in the "Cloud" section

**Comment by longzheng** [comment]
I can also confirm setting up an account with the country "Australia" leads to the same "Your account is not yet available for cloud development platform" error.

I worked around by creating a new account (with a new email) and set the country to "United States". Even though the account is also "Unverified identity", I'm able to access the Cloud projects screen.

**Comment by diskxo** [comment]
> Posso anche confermare che la creazione di un account con il paese "Australia" porta allo stesso errore "Il tuo account non è ancora disponibile per la piattaforma di sviluppo cloud".
> 
> Ho lavorato creando un nuovo account (con una nuova email) e impostando il paese su "Stati Uniti". Anche se l'account è anche "Identità non verificata", posso accedere alla schermata dei progetti Cloud.

Thanks, I will try in this mode

**Comment by codetheweb** [comment]
I updated the setup instructions: 23429f5.

**Comment by Kryd0s** [comment]
I just signed up for a new tuya.com account, and the same thing happened to me. I opened a support ticket, and they responded in about 3 hours. They asked one question, "Which country are you from?" I said "United States" and they authorized my account. That's it. Pretty simple process.

> I worked around by creating a new account (with a new email) and set the country to "United States". Even though the account is also "Unverified identity", I'm able to access the Cloud projects screen.

This no longer works. I selected United States before I opened the ticket, but couldn't get access on my own.
